src/editors/components/EditorHeader/HeaderTitle.test.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import * as module from './HeaderTitle';
+import { actions, selectors } from '../../data/redux';
+import { localTitleHooks } from './hooks';
+
+jest.mock('../../data/redux', () => ({
+  actions: {
+    app: {
+      setBlockTitle: jest.fn().mockName('actions.app.setBlockTitle'),
+    },
+  },
+  selectors: {
+    app: {
+      typeHeader: jest.fn(state => ({ typeHeader: state })),
+      isInitialized: jest.fn(state => ({ studioEndpointUrl: state })),
+    },
+  },
+}));
+
+jest.mock('./hooks', () => ({
+  localTitleHooks: jest.fn(),
+}));
+
+describe('HeaderTitle', () => {
+  const props = {
+    editorRef: jest.fn().mockName('args.editorRef'),
+    isInitialized: false,
+    setBlockTitle: jest.fn().mockName('args.setBlockTitle'),
+    typeHeader: 'html',
+  };
+  const hookProps = {
```

Comment:
nit: perhaps consider renaming `hookprops`. That name implies that it is props for either the component, or the hook, when it is in fact the return value of the hook-- something we often refer to as a `state`. Just a nit though.